types/bloom.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import "fmt"
+
+const (
+	// BloomByteLength represents the number of bytes used in a header log bloom.
+	BloomByteLength = 256
```

Comment:
We discussed this with Wasuwee when I deleted this, maybe for our internal types we shouldn't carry constants from geth. Anyway, it's not that important

Reply:
Some of them are required by Aida. Its a good call to only copy the ones we actually need.